README.md

```diff
@@ -138,6 +138,7 @@ our company and culture evolve, so should this document.
 - Take pride in how you build the software and your craftsmanship regardless of how the product does in the market. This doesn't mean gold-plating everything, often you should act pragmatically. Shipping value to users is the most important thing, but make sure to take the time to reflect on what you build, and do a post mortem and always make sure you're not
 repeating the mistakes of the past.
 - Ensure you challenge designs that you see problems with instead of implementing them without questioning.
+- Don't accept anything you find on the internet (StackOverflow, Github, etc) without understanding how it works. Vet all solutions against the API documentation and source code (when available). It may be the accepted answer, but that does not mean it's correct or that it is still current.
```

Comment:
Maybe another way to put this... Internet resources such as StackOverflow are valuable as a source of advice and example code, but we don't treat them as a source of truth. We always try to understand and validate what we find online before using it.

I prefer the use of we rather than you in general - it's an inclusive way to say this is what we all do, rather than this is what you should do - the latter can come over as didactic or even preachy.